src/olympia/tags/models.py

```diff
@@ -28,6 +28,13 @@ class Tag(ModelBase):
     class Meta:
         db_table = 'tags'
         ordering = ('tag_text',)
+        indexes = [
+            models.Index(fields=('denied', 'num_addons'),
+                         name='tag_blacklisted_num_addons_idx')
+        ]
+        constraints = [
+            models.UniqueConstraint(fields=('tag_text',), name='tag_text')
```

Comment:
should we be declaring `unique=True` in the column definition too for single column constraints?

Reply:
same question about the (non-fk) columns that aren't declared with `db_index=True` I suppose.

Reply:
It's not necessary if it's in `constraints`/`indexes`. In fact, at some point I ran into some issues when I had both and a custom name, I think django was confused and tried to add the index twice (I think it was with the `getpersonas_id` field, which is why I removed its `db_index` property)